src/main/java/com/yash/ExpenseClaims/controllers/HomeController.java

```diff
@@ -0,0 +1,31 @@
+package com.yash.ExpenseClaims.controllers;
+
+import org.springframework.security.core.Authentication;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.ModelMap;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.ModelAndView;
+import java.io.IOException;
+
+@Controller
+public class HomeController {
+
+    @GetMapping("/")
+    public ModelAndView homeView(Authentication authentication) throws IOException {
+        String userName = authentication.getName();
+        String newUserName = userName.substring(0, 1).toUpperCase() + userName.substring(1).toLowerCase();
```

Comment:
:x:  get Firstname and Lastname from database

Reply:
Currently, we are using spring security provided Users table schema which has username, password, enabled columns. So, we don't have the first name and last name fields in the database. However, our current business requirement needs the username (not first name and last name).

Reply:
so business requirement is to show camel cased Username :question:

Reply:
usernames are usually case-sensitive so why do we need to convert to camelCase :question:

Reply:
Yes, our business requirement needs the username to displayed in CamelCase. For example, if the username is "sankeerth", we need to display it as "Sankeerth".